service/collector/vault.go

```diff
@@ -61,9 +61,9 @@ func (v *Vault) Collect(ch chan<- prometheus.Metric) error {
 
 	secret, err := v.vaultClient.Auth().Token().LookupSelf()
 	if IsVaultAccess(err) {
-		v.logger.LogCtx(ctx, "level", "debug", "message", "vault not reachable")
-		v.logger.LogCtx(ctx, "level", "debug", "message", "vault upgrade in progress")
-		v.logger.LogCtx(ctx, "level", "debug", "message", "canceling collection")
+		v.logger.LogCtx(ctx, "level", "debug", "message", "vault not reachable")       // nolint: errcheck
+		v.logger.LogCtx(ctx, "level", "debug", "message", "vault upgrade in progress") // nolint: errcheck
+		v.logger.LogCtx(ctx, "level", "debug", "message", "canceling collection")      // nolint: errcheck
```

Comment:
This should not be necessary anymore with the latest `micrologger` changes. Same for all the other occurrences of that.

Reply:
Fixed.